src/app/_components/CrossIcon/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './CrossIcon';
```

Comment:
🔴 please stop using 2 files and 1 folder for each component, there is no point in doing so

Reply:
Thanks for comment! Done.